src/openrct2/ride/coaster/FlyingRollerCoaster.cpp

```diff
@@ -17506,6 +17489,60 @@ static void flying_rc_track_right_banked_25_deg_down_to_flat(
     flying_rc_track_flat_to_left_banked_25_deg_up(session, ride, trackSequence, (direction + 2) & 3, height, trackElement);
 }
 
+// Exact copy of what's found in BolligerMabillardTrack.hpp as I cannot include the file because then the project won't build.
```

Comment:
Not acceptable, I’m afraid. We’ll have to find another solution.

Reply:
Sadly this is the only thing that the include was pulling at this moment in time from that file. It's why I did it like this. 
I don't exactly know what's going on, but it seems like the flying rc doesn't appreciate a regular inline twist being in the include. That's error it kept giving me.